tests/test_selenium_app.py

```diff
@@ -0,0 +1,136 @@
+import os
+import random
+from threading import Thread
+
+import pytest
+from dotenv import find_dotenv, load_dotenv
+from selenium import webdriver
+
+import app
+import common
+from trello_request import TrelloRequest
+
+
+class TrelloCreateBoard(TrelloRequest):
+    URL_PATH = "/boards"
+
+    def create(self, board_name):
+        url = self.URL_PATH
+
+        query = {
+            'name': board_name
+        }
+
+        json_data = super().makeRequest(url, "POST", query)
+
+        return json_data['id']
+
+
+class TrelloDeleteBoard(TrelloRequest):
+    URL_PATH = "/boards/{}"
+
+    def delete(self, board_id):
+        url = self.URL_PATH.format(board_id)
+        return super().makeRequest(url, "DELETE")
+
+
+class TrelloCreateList(TrelloRequest):
+    URL_PATH = "/lists"
+
+    def create(self, name, board_id):
+        url = self.URL_PATH
+
+        query = {
+            'name': name,
+            'idBoard': board_id
+        }
+
+        json_data = super().makeRequest(url, "POST", query)
+
+        return json_data['id']
+
+
+def create_trello_board():
+    name = f"Selenium Test Board {random.randint(100, 1000)}"
+    return TrelloCreateBoard().create(name)
+
+
+def delete_trello_board(board_id):
+    return TrelloDeleteBoard().delete(board_id)
+
+
+def createList(board_id, list_name):
+    return TrelloCreateList().create(list_name, board_id)
+
+
+@pytest.fixture(scope='module')
+def test_app():
+    file_path = find_dotenv('.env')
+    load_dotenv(file_path, override=True)
+
+    # some load or setup issues with environment
```

Comment:
Nice spot here - because the key/token are set as class variables then they are defined when the file is imported which is _before_ you load the .env file above. This is one way we can fix that, alternatives might be that TrelloRequest sets these as instance variables (e.g. inside the `__init__` function), or that they're loaded from the environment in the `makeRequest` function directly